newswires/client/src/queryHelpers.ts

```diff
@@ -0,0 +1,27 @@
+import { presetIsInSports, topLevelSportId } from './presets.ts';
+import type { Query } from './sharedTypes.ts';
+
+export const removeValueFromQuery = (
```

Comment:
Given the name of this function, I think I'd expect it to return a `Query`, rather than `Partial<Query>`. I think it would also make it a bit easier to reason about at the call site too, rather than having to spread it alongside the original query -- what do you think?

Reply:
Yeah, I see what you mean. It's just then the output is depentent on the input which could have a bunch of different variations depending on what is already set in the Query object on input. 

This way we only affect the specific field that is being changed and return that value. 

It's maybe a bit overkill on technical purity, as this might just be easier to think about. I just feel wary of methods which depend on a complex input object for its output